consumer/src/test/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/saker/kilder/GsakSakerTest.kt

```diff
@@ -0,0 +1,194 @@
+
+    @Mock
+    private val sakApiGateway: SakApiGateway? = null
+
+
+    @InjectMocks
+    private val sakerService: SakerServiceImpl? = null
+    private var sakerListe: List<SakDto>? = null
+
+    @BeforeEach
+    @Throws(FinnSakUgyldigInput::class, FinnSakForMangeForekomster::class)
+    fun setUp() {
+        EnvironmentUtils.setProperty("SAK_ENDPOINTURL", "https://sak-url", EnvironmentUtils.Type.PUBLIC)
+        MockitoAnnotations.initMocks(this)
+        sakerService!!.setup() // Kaller @PostConstruct manuelt siden vi kjører testen uten spring
+        sakerListe = createSaksliste()
+        Mockito.`when`(sakApiGateway!!.hentSaker(any())).thenReturn(sakerListe)
+        Mockito.`when`(arbeidOgAktivitet!!.hentSakListe(ArgumentMatchers.any(WSHentSakListeRequest::class.java))).thenReturn(WSHentSakListeResponse())
+    }
+
+    @Test
+    fun transformasjonenGenerererRelevanteFelter() {
+        val sakDto = SakDto(id = SakId_1,
+                tema = "AAP",
+                applikasjon = "PP01",
+                aktoerId = "123",
+                orgnr = null,
+                fagsakNr = null,
+                opprettetAv = null,
+                opprettetTidspunkt = FIRE_DAGER_SIDEN)
+        val sak = GsakSaker.TIL_SAK.invoke(sakDto)
+        MatcherAssert.assertThat(sak.saksId, Is.`is`(SakId_1))
+        MatcherAssert.assertThat(sak.fagsystemSaksId, Is.`is`(nullValue()))
+        MatcherAssert.assertThat(sak.temaKode, Is.`is`(Sak.GODKJENTE_TEMA_FOR_GENERELL_SAK[0]))
+        MatcherAssert.assertThat(sak.sakstype, Is.`is`(Sak.SAKSTYPE_GENERELL))
+        MatcherAssert.assertThat(sak.fagsystemKode, Is.`is`(Sak.FAGSYSTEMKODE_PSAK))
+        MatcherAssert.assertThat(sak.opprettetDato, Is.`is`(FIRE_DAGER_SIDEN))
+        MatcherAssert.assertThat(sak.finnesIGsak, Is.`is`(true))
+    }
+
+    @Test
+    fun `transformasjonen BrukerSaksId For FagsystemId Og MFS Som Sakstype Om Fagsystem Er Vedtakslosningen`() {
+        val sakDto = SakDto(id = SakId_1,
+                tema = "AAP",
+                applikasjon = VEDTAKSLOSNINGEN,
+                aktoerId = "123",
+                orgnr = null,
+                fagsakNr = FagsystemSakId_1,
+                opprettetAv = null,
+                opprettetTidspunkt = FIRE_DAGER_SIDEN)
+
+        val sak = GsakSaker.TIL_SAK.invoke(sakDto)
+        MatcherAssert.assertThat(sak.saksId, Is.`is`(SakId_1))
+        MatcherAssert.assertThat(sak.fagsystemSaksId, Is.`is`(SakId_1))
+        MatcherAssert.assertThat(sak.temaKode, Is.`is`(Sak.GODKJENTE_TEMA_FOR_GENERELL_SAK[0]))
+        MatcherAssert.assertThat(sak.sakstype, Is.`is`(Sak.SAKSTYPE_MED_FAGSAK))
+        MatcherAssert.assertThat(sak.fagsystemKode, Is.`is`(VEDTAKSLOSNINGEN))
+        MatcherAssert.assertThat(sak.opprettetDato, Is.`is`(FIRE_DAGER_SIDEN))
+        MatcherAssert.assertThat(sak.finnesIGsak, Is.`is`(true))
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun knytterBehandlingsKjedeTilSakUavhengigOmDenFinnesIGsak() {
```

Comment:
Syns vi skal samle oss rundt en måte å skrive funksjonsnavn på innenfor en fil. 
Så om testene bruker back-tick syntaksen ett sted så syns jeg vi bør gjøre det for de andre testene også.

Reply:
enig med back-tick syntaksen